XamarinCommunityToolkit/Views/MediaElement/MediaElement.ios.cs

```diff
@@ -164,7 +180,9 @@ void RemoveStatusObserver()
 				try
 				{
 					avPlayerViewController?.Player?.CurrentItem?.RemoveObserver(statusObserver, "status");
+					statusObserver.Dispose();
 				}
+				catch { }
```

Comment:
Hi, I doubt that such statements are a good idea. Catching all exceptions and do nothing, I mean.

Reply:
The thing is it will throw exceptions when the observer was removed for some reason. I got a few crashing reports regarding this, but I have no idea how it happens.

But if we didn't catch the exception from the PCL level there's no way to catch the exceptions and the App will crash.